struqture/tests/fermions/fermionic_open_system.rs

```diff
@@ -62,11 +62,38 @@ fn group() {
     assert_eq!(slos, FermionLindbladOpenSystem::default())
 }
 
+#[test]
+fn group_with_none() {
+    let slos = FermionLindbladOpenSystem::group(
```

Comment:
Probably not important, but maybe we want to call this `flos` instead

Reply:
Fixed